src/main/java/frc/robot/poseestimator/helpers/objectListOrganizer.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.poseestimator.helpers;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Supplier;
+
+public class objectListOrganizer<T> {
```

Comment:
How does it organize the list? Does it sort it from highest value to lowest?

Reply:
sorry i must have acsedentaly resolved it instead of something else

Reply:
gill allready oferd an alternative name for this class:ObservationAccumulator, although it might need some adjusting because it does'nt only get observetion